Include/internal/pycore_debug_offsets.h

```diff
@@ -23,7 +23,7 @@ extern "C" {
    declaration                                      \
    _GENERATE_DEBUG_SECTION_LINUX(name)
 
-#if defined(MS_WINDOWS)
+#if defined(MS_WINDOWS) && !defined(__clang__)
```

Comment:
Does this need an alternate definition for Windows + clang?

Reply:
No, below there's a default that it uses is we're not one of the supported platforms. And that default works for Clang Windows.